src/Core/Framework/Adapter/Asset/ThemeAssetPackage.php

```diff
@@ -44,6 +43,11 @@ public function getUrl(string $path)
         return $this->getBaseUrl($path) . $this->appendThemePath() . $url;
     }
 
```

Comment:
Revert that pls. I will fix it on my own mr already working on the theme system

Reply:
Dropped the commit